pkg/xds/server/grpc.go

```diff
@@ -33,7 +33,7 @@ func (s *grpcServer) Start(stop <-chan struct{}) error {
 	grpcOptions = append(grpcOptions, grpc.MaxConcurrentStreams(grpcMaxConcurrentStreams))
 	grpcServer := grpc.NewServer(grpcOptions...)
 
-	lis, err := net.Listen("tcp", fmt.Sprintf(":%d", s.port))
+	lis, err := net.Listen("tcp", fmt.Sprintf("127.0.0.1:%d", s.port))
```

Comment:
Please revert this back so the server can be accessed outside of the control plane machine.